src/sources/socket/tcp.rs

```diff
@@ -174,7 +179,10 @@ impl TcpSource for RawTcpSource {
                     .as_deref()
                     .unwrap_or_else(|| log_schema().host_key());
 
-                log.try_insert(host_key, host.clone());
+                log.try_insert(host_key, host.ip().to_string());
+                if let Some(port_key) = &self.config.port_key {
+                    log.try_insert(port_key.as_str(), host.port().to_string());
```

Comment:
Ah here is where my comment is actually relevant.
Shouldn't be this inserted as a 16-bit integer instead of a string?

Reply:
Yes, you are right 😄